source/configuration_manual/replication.rst

```diff
@@ -0,0 +1,415 @@
+.. _replication:
+
+======================
+Replication with dsync
+======================
+
+.. note::
+
+  This is not supported with Dovecot Pro.
+
+Dovecot supports master/master replication using dsync. It's recommended
```

Comment:
Could the first sentence be "It is possible to enable master/master replication using dsync."?